src/styles/sidebar.css

```diff
@@ -1,29 +1,66 @@
 .sidebar {
-  background-color: var(--LIGHT-NAV-BG);
+  background-color: var(--CARD-LIGHT);
   display: flex;
   flex-direction: column;
-  gap: 1rem;
+  gap: 2rem;
   position: fixed;
   top: 3rem;
   height: 95vh;
-  width: fit-content;
-  min-width: 10rem;
-  padding: 2rem 1rem 0 10px;
+  min-width: 5rem;
+  padding: 2rem 1rem 2rem 10px;
 }
 
 .sidebar.dark {
-  background-color: var(--DARK-NAV-BG);
+  background-color: var(--DARK-THEME-BG);
 }
 
-.sidebar.dark .sidebar-links {
+.dark .sidebar-links {
   color: var(--CTAS-DARK);
 }
 
 .sidebar-links {
   display: flex;
-  gap: 1rem;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 5px;
+  height: 5rem;
 }
 
 .sidebar-icons {
   font-size: 24px;
 }
+
+.sidebar-links.active {
+  background-color: var(--CTAS-DARK);
+  color: black;
+  border-radius: 1rem;
+}
+
+@media (max-width: 810px) {
+  .sidebar {
+    display: flex;
+    flex-direction: row;
+    top: auto;
+    bottom: 0px;
+    min-width: 100%;
+    z-index: 1;
+    justify-content: space-around;
+    height: fit-content;
+    padding: 0px;
+    gap: 1rem;
+  }
+
+  .sidebar-pages {
+    display: none;
+  }
+
+  .sidebar-links {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
```

Comment:
Suggestion: you can make a utility class `flex-column-center` in order to avoid writing same piece of code